problems/oldestPerson.js

```diff
@@ -12,7 +12,12 @@
  * returns: 'Kevin'
  */
 function oldestPerson() {
-
-}
+    let obj = {};
+    let person = []
+    for (let i = 0; i <= person.length; i++) {
+        return (obj.filter(i) + "")
```

Comment:
this is not a proper use if filter, we don't really want to filter out anything we just want to get the biggest number that we find within an object.